aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -0,0 +1,110 @@
+#include "ATen/ATen.h"
+#include "ATen/Dispatch.h"
+#include "ATen/ExpandUtils.h"
+#include "ATen/NativeFunctions.h"
+#include "ATen/WrapDimUtils.h"
+#include "cpu/UnaryOpsKernel.h"
+
+#include <algorithm>
+#include <functional>
+#include <numeric>
+#include <vector>
+
+#include <map>
+
+namespace at {
+namespace native {
+
+using unary_type = void(Tensor &, const Tensor &);
+unary_type *ceilImpl = DispatchStub<unary_type>::init<ceilImplC, &ceilImpl>;
```

Comment:
Stick to the style used in other native functions: e.g. `type* name` instead of `type *name`

Reply:
I'll run clang-format at the end.